### PR TITLE
Timeline division and more ordering options

### PR DESCRIPTION
**Pull request by w568w**
Treat `division id = 0` as a "timeline" division.

Also, make the backend able to return a hole list ordered by a different key.

See file changes for more detail.

### CODE REVIEW

File: api/api.py [hunk 1]
```diff
@@ -263,6 +263,13 @@ def get(self, request, **kwargs):
         prefetch_length = serializer.validated_data.get('prefetch_length')
         start_time = serializer.validated_data.get('start_time')
         division_id = serializer.validated_data.get('division_id')
+
+        # 返回结果的排序
+        # @w568w (2022/3/30): 我对 Serializer 的组织方式并不熟悉，因此采用了这种比较原始的参数读取方法。
+        order = request.query_params['order']
+        if not order:
+            order = 'time_updated'
```
Comment (hasbai) [inline api/api.py:271]: order = request.query_params.get('order', 'time_updated')，否则 key error

**Comment by hasbai** [comment]
建议本地跑过测试后再提交

**Comment by w568w** [comment]
> 建议本地跑过测试后再提交

OK。但是我看你也这么写的……
https://github.com/OpenTreeHole/backend/blob/4ff9697badd5625efc199dba9d0138a37aaa4d0d/api/api.py#L119

File: api/api.py (hunk 1 above)
Comment (w568w) [reply api/api.py]: OK, fixed.

**Pull request merged**